### DISCUSSION TITLE
Error in Fabric usage (WSL) -- extract_wisdom Pattern

### DISCUSSION THREAD

**Discussion by arkay55** [danielmiessler/Fabric#487] [General]
After setup Fabric on WSL (Windows 11), I am unable to run the extract_wisdom Pattern with the --stream option to get immediate and streaming results from any Youtube video

Get the following error:
![Fabric_error1](https://github.com/danielmiessler/fabric/assets/171407645/372028f2-d636-4833-b247-85c8c577d5a8)

Newbie here, but really excited for this technology of summarizing a youtube video!
What else am I missing? 

Thanks!

**Comment by black-backdoor** [comment]
Try running the `fabric --listmodels` command. This will list all the available models. Make sure gpt-4-turbo appears in the list.

If it’s not listed, you might need to configure it properly. Let us know how it goes!

**Reply by arkay55** [reply]
gpt-4-turbo does not appears in the list, do I need paid subscription to use it? if not, please provide the configuration steps

**Reply by blubomber** [reply]
I ran into a similar issue. I had to go here and log into my openai account. https://platform.openai.com/settings/organization/limits. Its the same area you get your API key. But you have purchase a token package. here is the information on the packages. https://platform.openai.com/settings/organization/limits I purchased the Tier 1 which is not enough for a long youtube video transcript but is good for smaller text. Hope that helps.
